cpp/ql/src/experimental/Security/CWE/CWE-552/DangerousUseRename.qhelp

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>If the rename function did not work correctly, simply overwriting the destination file can lead to errors. Use access to the destination file with permission setting.</p>
```

Comment:
What kind of errors can occur?

Reply:
arbitrary files can be deleted on behalf of a running program.

Reply:
This needs to be mentioned here.

Reply:
I corrected the inscription in qhelp, how do you like it?